src/players/dto/player.dto.ts

```diff
@@ -0,0 +1,17 @@
+export interface PlayerDto {
+  id: string;
+  name: string;
+  steamId: string;
+  joinedAt: Date;
+  avatar: {
+    small?: string;
+    medium?: string;
+    large?: string;
+  };
+  roles: ('super user' | 'admin' | 'bot')[];
```

Comment:
again with enum- maybe this should be done in a follow-up instead?

Reply:
I'd like to avoid using enums in DTO - it's something that should be easy for any API consumers to read.

Reply:
Not sure if that's a good argument, tbh I think `Role[]` is much more readable.

Reply:
Yeah, but you're forcing the user to lookup the `Role` type, which is not a part of the DTO.